code/modules/mob/living/status_procs.dm

```diff
@@ -0,0 +1,522 @@
+		jitteriness = 0
+
+/mob/living/AdjustJitter(amount, bound_lower = 0, bound_upper = INFINITY, force = 0)
+	var/new_value = directional_bounded_sum(jitteriness, amount, bound_lower, bound_upper)
+	SetJitter(new_value, force)
+
+// LOSE_BREATH
+
+/mob/living/LoseBreath(amount)
+	SetLoseBreath(max(lose_breath, amount))
+
+/mob/living/SetLoseBreath(amount)
+	lose_breath = max(amount, 0)
+
+/mob/living/AdjustLoseBreath(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(lose_breath, amount, bound_lower, bound_upper)
+	SetLoseBreath(new_value)
+
+// PARALYSE
+
+/mob/living/Paralyse(amount, updating = 1, force = 0)
+	SetParalysis(max(paralysis, amount), updating, force)
+
+/mob/living/SetParalysis(amount, updating = 1, force = 0)
+	if(status_flags & CANPARALYSE || force)
+		paralysis = max(amount, 0)
+	else
+		paralysis = 0
+	if(updating)
+		update_canmove()
+		update_stat()
+
+/mob/living/AdjustParalysis(amount, bound_lower = 0, bound_upper = INFINITY, updating = 1, force = 0)
+	var/new_value = directional_bounded_sum(paralysis, amount, bound_lower, bound_upper)
+	SetParalysis(new_value, updating, force)
+
+// SILENT
+
+/mob/living/Silence(amount)
+	SetSilence(max(silent, amount))
+
+/mob/living/SetSilence(amount)
+	silent = max(amount, 0)
+
+/mob/living/AdjustSilence(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(silent, amount, bound_lower, bound_upper)
+	SetSilence(new_value)
+
+// SLEEPING
+
+/mob/living/Sleeping(amount, updating = 1)
+	SetSleeping(max(sleeping, amount), updating)
+
+/mob/living/SetSleeping(amount, updating = 1)
+	sleeping = max(amount, 0)
+	update_sleeping_effects()
+	if(updating)
+		update_stat()
+
+/mob/living/AdjustSleeping(amount, bound_lower = 0, bound_upper = INFINITY, updating = 1)
+	var/new_value = directional_bounded_sum(sleeping, amount, bound_lower, bound_upper)
+	SetSleeping(new_value, updating)
+
+// SLOWED
+
+/mob/living/Slowed(amount, updating = 1)
+	SetSlowed(max(slowed, amount), updating)
+
+/mob/living/SetSlowed(amount, updating = 1)
+	slowed = max(amount, 0)
+
+/mob/living/AdjustSlowed(amount, bound_lower = 0, bound_upper = INFINITY, updating = 1)
+	var/new_value = directional_bounded_sum(slowed, amount, bound_lower, bound_upper)
+	SetSlowed(new_value, updating)
+
+// SLURRING
+
+/mob/living/Slur(amount)
+	SetSlur(max(slurring, amount))
+
+/mob/living/SetSlur(amount)
+	slurring = max(amount, 0)
+
+/mob/living/AdjustSlur(amount, bound_lower = 0, bound_upper = INFINITY)
+	var/new_value = directional_bounded_sum(slurring, amount, bound_lower, bound_upper)
+	SetSlur(new_value)
+
+// STUN
+
+/mob/living/Stun(amount, updating = 1, force = 0)
+	SetStunned(max(stunned, amount), updating, force)
+
+/mob/living/SetStunned(amount, updating = 1, force = 0) //if you REALLY need to set stun to a set amount without the whole "can't go below current stunned"
+	if(status_flags & CANSTUN || force)
+		stunned = max(amount, 0)
+	else if(stunned)
```

Comment:
This else is going to cause issues.

The whole reason `force` exists is to *force* stun/weaken/paralysis immune things to be incapped no matter what...Xenos use this, for example---if this `else` is here...then a xeno that is force stunned for say...5 cycles will only ever be stunned for 1 cycle, because `Life` will tick and it'll see they can't be stunned and instantly set paralysis to 0.